packages/vats/test/bootstrapTests/test-liquidation-1.js

```diff
@@ -0,0 +1,374 @@
+    const offerId = `open-vault${i}`;
+    await minter.executeOfferMaker(Offers.vaults.OpenVault, {
+      offerId,
+      collateralBrandKey,
+      wantMinted: setup.vaults[i].ist,
+      giveCollateral: setup.vaults[i].atom,
+    });
+    t.like(minter.getLatestUpdateRecord(), {
+      updated: 'offerStatus',
+      status: { id: offerId, numWantsSatisfied: 1 },
+    });
+  }
+
+  // Verify starting balances
+  for (let i = 0; i < setup.vaults.length; i += 1) {
+    check.vaultNotification(i, {
+      debtSnapshot: { debt: { value: scale6(setup.vaults[i].debt) } },
+      locked: { value: scale6(setup.vaults[i].atom) },
+      vaultState: 'active',
+    });
+  }
+
+  const buyer = await walletFactoryDriver.provideSmartWallet('agoric1buyer');
+  {
+    // ---------------
+    //  Place bids
+    // ---------------
+
+    const parseAmount = makeParseAmount(agoricNamesRemotes, Error);
+    await buyer.sendOffer(
+      Offers.psm.swap(
+        agoricNamesRemotes.instance['psm-IST-USDC_axl'],
+        agoricNamesRemotes.brand,
+        {
+          offerId: 'print-ist',
+          wantMinted: 1_000,
+          pair: ['IST', 'USDC_axl'],
+        },
+      ),
+    );
+
+    const maxBuy = '10000ATOM';
+
+    // bids are long-lasting offers so we can't wait here for completion
+    await buyer.sendOfferMaker(Offers.auction.Bid, {
+      offerId: 'bid1',
+      ...setup.bids[0],
+      maxBuy,
+      parseAmount,
+    });
+
+    t.like(readLatest('published.wallet.agoric1buyer'), {
+      status: {
+        id: 'bid1',
+        result: 'Your bid has been accepted',
+        payouts: undefined,
+      },
+    });
+    await buyer.sendOfferMaker(Offers.auction.Bid, {
+      offerId: 'bid2',
+      ...setup.bids[1],
+      maxBuy,
+      parseAmount,
+    });
+    t.like(readLatest('published.wallet.agoric1buyer'), {
+      status: {
+        id: 'bid2',
+        result: 'Your bid has been accepted',
+        payouts: undefined,
+      },
+    });
+    await buyer.sendOfferMaker(Offers.auction.Bid, {
+      offerId: 'bid3',
+      ...setup.bids[2],
+      maxBuy,
+      parseAmount,
+    });
+    t.like(readLatest('published.wallet.agoric1buyer'), {
+      status: {
+        id: 'bid3',
+        result: 'Your bid has been accepted',
+        payouts: undefined,
+      },
+    });
+  }
+
+  {
+    // ---------------
+    //  Change price to trigger liquidation
+    // ---------------
+
+    await priceFeedDriver.setPrice(9.99);
+
+    // check nothing liquidating yet
+    /** @type {import('@agoric/inter-protocol/src/auction/scheduler.js').ScheduleNotification} */
+    const liveSchedule = readLatest('published.auction.schedule');
+    t.is(liveSchedule.activeStartTime, null);
+    t.like(readLatest('published.vaultFactory.managers.manager0.metrics'), {
+      numActiveVaults: setup.vaults.length,
+      numLiquidatingVaults: 0,
+    });
+
+    // advance time to start an auction
+    console.log('step 0 of 10');
```

Comment:
Can these `console.log()` messages be dropped or converted to inactive `trace()`? I can see how they'd be helpful while building and debugging the test, but their pretty noisy after that.

Reply:
ok, changed all logs to trace, disabled